docs/modules/user-guide/partials/proc_developing-with-devfiles.adoc

```diff
@@ -0,0 +1,36 @@
+[id="proc_developing-with-devfiles_{context}"]
+= Developing with devfiles
+[role="_abstract"]
+
+Use the devfile specification to develop a node.js “Hello World” application. Developing this application introduces you to how a devfile automates and simplifies your development process.
+
+.Prerequisites
```

Comment:
I assume this Prereqs format is consistent with the rest of devfile docs and looks good among them?

I mean a somewhat "more expected" Prereq list would look like:
- A Kubenertes cluster is installed on your local machine. To...
- Kubectl is installed...
etc.
But I do like your more explanatory Prereq section with why users need this or that step.